packages/platform-sdk-btc/src/services/message.ts

```diff
@@ -23,7 +23,7 @@ export class MessageService implements Contracts.MessageService {
 
 	public async sign(input: Contracts.MessageInput): Promise<Contracts.SignedMessage> {
 		try {
-			const mnemonic: string = BIP39.normalize(input.mnemonic);
+			const mnemonic: string = BIP39.normalize(input.mnemonic!);
```

Comment:
This needs a proper check that the mnemonic is set now that it isn't enforced by TypeScript.